3rdparty/ErrorHandling/errorcodes.h

```diff
@@ -0,0 +1,29 @@
+#include <map>
+
+struct ErrorCodes
+{
+    enum Errors {
+        UnsupportedFormat,
+        FileNotFound,
+        Blank,
+        ParsemzXml,
+        ParsemzMl,
+        ParsemzData,
+        ParseCsv
+    };
+
+};
+
+//singleton
+class ErrorMsg
```

Comment:
@rish9511 Class name and file should be same.

Reply:
Yes.
should I add this to TODO  or do you want me to change it now ?